recipes/portalocker/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "portalocker" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "b955d6197c7cd3206500d3237a2dd6cc899ceebb13460cc97e658cbab70df11b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - python
+    - pytest-runner
+
+  run:
+    - python
+
+test:
+  imports:
+    - portalocker
+
+about:
+  home: http://portalocker.readthedocs.org/en/latest/
+  license: PSF
+  summary: 'Portalocker is a library to provide an easy API to file locking.'
+
+extra:
+  recipe-maintainers:
```

Comment:
@johanneskoester and @bgruening -- are you ok being listed as co-maintainers of the conda-forge recipes for `portalocker` and `sompy`?